packages/web-components/src/components/va-button/va-button.scss

```diff
@@ -17,22 +17,13 @@
 }
 
 // Fix to off-centered content in V3 when icon present
```

Comment:
I see some CSS deleted. Do you need this comment here still?

Reply:
Yes, that comment still applies to the remaining CSS.